src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,126 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private int randomInt;
+
+    public int randomNumberForLegs() {

```

Comment:
1. FigureSupplier should have only 2 public methods: getRandomFigure() and getDefaultFigure()
2. Public methods should go first and private methods after them

Reply:
++